mdal/frmts/mdal_3di.cpp

```diff
@@ -121,10 +121,10 @@ void MDAL::Driver3Di::addBedElevation( MemoryMesh *mesh )
   size_t faceCount = mesh->facesCount();
 
   // read Z coordinate of 3di computation nodes centers
-  int ncidZ = mNcFile.getVarId( "Mesh2DFace_zcc" );
-  double fillZ = mNcFile.getFillValue( ncidZ );
+  int ncidZ = mNcFile->getVarId( "Mesh2DFace_zcc" );
+  double fillZ = mNcFile->getFillValue( ncidZ );
   std::vector<double> coordZ( faceCount );
-  if ( nc_get_var_double( mNcFile.handle(), ncidZ, coordZ.data() ) )
+  if ( nc_get_var_double( mNcFile->handle(), ncidZ, coordZ.data() ) )
```

Comment:
No Exception throw for this fail is needed?

Reply:
probably not